files/scripts/gbsyncd.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+SERVICE="gbsyncd"
+PEER="swss"
+DEBUGLOG="/tmp/swss-gbsyncd-debug.log"
+LOCKFILE="/tmp/swss-gbsyncd-lock"
```

Comment:
where does this file come from? is it from syncd.sh? can we modify the syncd.sh to pass parameter, it is better to maintain one file instead of multiple files.

Reply:
add a syncd_common.sh for holding common functions between them.

Reply:
Done